mmv1/third_party/terraform/tests/resource_container_cluster_test.go.erb

```diff
@@ -1015,21 +1015,21 @@ func TestAccContainerCluster_withSandboxConfig(t *testing.T) {
 		},
 	})
 }
+<% end -%>
 
 func TestAccContainerCluster_withBootDiskKmsKey(t *testing.T) {
-	// KMS key has calculated next rotation time
-	skipIfVcr(t)
```

Comment:
Is this no longer necessary? What changed? Or was it unnecessary in the first place?

Reply:
It looks like this is obviated by `BootstrapKMSKeyInLocation`.

Reply:
Yeah, the test is more efficient now by reusing bootstrap kms keys and making VCR usable